src/main/java/com/jhuguet/sb_taskv1/app/web/security/JwtFilter.java

```diff
@@ -34,7 +34,7 @@ public JwtFilter(CustomUserDetailsServiceImpl detailsService) {
     @Override
     protected boolean shouldNotFilter(HttpServletRequest request) {
         String path = request.getServletPath();
```

Comment:
Let's create an array of mappings, that shouldn't be filtered and then use arr.contains(url)